pkgs/applications/editors/vim/plugins/sg-nvim/default.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  vimPlugins,
+  vimUtils,
+}:
+let
+  version = "1.1.0-unstable-2024-12-15";
+  src = fetchFromGitHub {
+    owner = "sourcegraph";
+    repo = "sg.nvim";
+    rev = "7c423ebeb028b3534eb20fe54972825384dbe7d6";
+    sha256 = "1d9ag1q9yxiai5hfm0x4am710gqzx3liw4qg10glskjczmgiic00";
```

```suggestion
    hash = "sha256-ALAYX/1MTk0fCA8THunoHz8QTlWkg+pgiSp2n3B4KrU=";
```

`nix-update` doesn't know how to properly handle `sha256 = ...` and updates it to

```nix
sha256 = "sha256-ALAYX/1MTk0fCA8THunoHz8QTlWkg+pgiSp2n3B4KrU=";
```

Though this works fine

---

Thanks for catching that.